README.md

```diff
@@ -1,8 +1,8 @@
 # Ramps
 
-Ramps allow lend and pay to be made with any ERC-20 token while the Diaspore contract receiving RCN their token using pooled liquidity.
+Ramps allow lend and pay to be made with any ERC-20 token while the protocol receiving RCN their token.
 
-TokenConverter accepts tokens and ether, converts these to the RCN token, and makes corresponding call.
+TokenConverter accepts tokens and ether, converts these to the RCN token, and makes corresponding calls.
```

Comment:
TokenConverter accepts tokens and ether, and converts these to the RCN token, making the corresponding calls.